clang/docs/StandardCPlusPlusModules.rst

```diff
@@ -477,21 +476,21 @@ When the ``-fmodules-embed-all-files`` flag are enabled, Clang explicitly emits
 code into the BMI file, the contents of the BMI file contain a sufficiently verbose
 representation to reproduce the original source file.
 
-:sup:`1`` Input files: The source files which took part in the compilation of the BMI.
-For example:
+.. [1] Input files: The source files which took part in the compilation of the BMI.
```

Comment:
I am not familiar with RST. Are you tested with this?

Reply:
Here is how it looks like:
![2025-04-23T09:13:50,035090911+02:00](https://github.com/user-attachments/assets/3e096ff3-6dbe-4848-8b1c-b79de555275e)

And when clicked:
![2025-04-23T09:14:08,105023675+02:00](https://github.com/user-attachments/assets/dd1a1b97-7282-4430-93fd-c65ec815d8b0)

Reply:
Fine enough. Thanks.